onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -1037,6 +1037,14 @@ MlasConvertHalfToFloatBuffer(
     size_t Count
     );
 
+extern "C" void
+MLASCALL
+MlasConvertHalfToFloatBufferAVX2(
```

Comment:
as we discussed, could you please combine the MlasConvertHalfToFloatBufferAVX2 and MlasConvertHalfToFloatBuffer function?

Reply:
Done!